recipes/example/ispyb/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "ispyb" %}
+{% set version = "4.1.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "c1f500301de5abfc37f28c88673d5970064d003e73dd5b6a95433292b07adfb3" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - enum34  # [py27]
+    - pip
+    - mysql-connector-python
+    - pytest-runner
```

Comment:
Is this a build requirement?

Reply:
Strangely, yes it is.  Without it I can't build locally!

Reply:
Hmm that is odd. I presume the others are in the same boat.

Reply:
For some reason, my local env was messed up - I think I can remove `enum34` and `mysql-connector-python`